content/en/1.13.x/concepts/updating-bottlerocket.markdown

```diff
@@ -0,0 +1,299 @@
+# Updating Bottlerocket
+
+This document covers the different ways to update Bottlerocket (for [ECS](#ecs), [Kubernetes](#kubernetes), and [SSM](#ssm)), as well as how to [lock your nodes to a specific release](#locking-to-a-specific-release).
```

Comment:
"document" is odd here - it's probably not going to be consumed solo. Maybe "There are many ways to update Bottlerocket including..." or something similar

Reply:
Addressed in commmit https://github.com/bottlerocket-os/project-website/pull/28/commits/db0f57f6c796b2a8d0565cf899a78fc85af84fe5